### PR TITLE
Update large-dataset-datacap-application.yaml

### PR DESCRIPTION
**Pull request by simonkim0515**
2 new questions added to LDN app

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/large-dataset-datacap-application.yaml [hunk 1]
```diff
@@ -344,6 +343,27 @@ body:
       - More Than 15PiB
   validations:
     required: true
+- type: input
+  attributes:
+    label: What is the size of your dataset (One Copy)
```
Comment (galen-mcandrew) [inline .github/ISSUE_TEMPLATE/large-dataset-datacap-application.yaml:348]: Looks like other prompts are not questions (such as Number of replicas). Should we reword this to a statement, such as: "Expected size of single dataset (one copy)"

File: .github/ISSUE_TEMPLATE/large-dataset-datacap-application.yaml [hunk 2]
```diff
@@ -344,6 +343,27 @@ body:
...
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Number of replicas required
```
Comment (galen-mcandrew) [inline .github/ISSUE_TEMPLATE/large-dataset-datacap-application.yaml:353]: Required? Requested? Desired? Maybe just: Number of replicas to store

**Review by galen-mcandrew — COMMENTED** [review]
Added two comments, I like this direction! Excited to get more granularity in client onboarding.

**Comment by data-programs** [comment]
KYC

This user’s identity has been verified through [filplus.storage](https://filplus.storage)
